watertap/data/techno_economic/gac.yaml

```diff
@@ -1,4 +1,7 @@
 default: # TODO: confirm any differentiation in removal across subtypes
+  volumetric_electricity_intensity:
+    value: 0.106310
+    units: kW/m^3
```

Comment:
This is technically power intensity if we have units of kW/m3 (whereas we've been using electricity intensity for energy consumption per volumetric flowrate at inlet in units of kWh/m3).

Reply:
Although I see why you called it this-- dividing by volume rather than volume/time.

Reply:
I am not sure that is quite right either. For this unit, we have `intensity = a*ebct` where this is `a`. I.e. this is a parameter for calculating intensity based on the empty bed contact time.

Reply:
Actually, doesn't multiplying by `ebct` turn intensity into kWh/m3  (i.e., multiplying by time) rather than canceling out time units?

Reply:
No, intensity has units of kWh/m^3 (at least for every other model we have done).

Reply:
Yes, that's what I am saying too-- we'll end up with kWh/m^3, which is correct. 

To clarify my point, I think a better name for `volumetric_electricity_intensity` would be `power_electric_flow_vol_inlet` (analogous to our naming for `energy_electric_flow_vol_inlet`, where energy is kWh, and in `power_electric_flow_vol_inlet`, power is kW).